packs/data/extinction-curse-bestiary.db/doblagub.json

```diff
@@ -1690,7 +1960,7 @@
                     "value": ""
                 },
                 "attackEffects": {
-                    "custom": "",
+                    "custom": "Pull 10 ft",
                     "value": [
                         "grab",
                         "magical-tongue"
```

Comment:
comment actually applies to line 1972 - should this non-damaging attack have a damageRolls entry?

Reply:
A new build will probably remake it, but I have removed.